### ISSUE TITLE
[a11y] Automatic level badge has poor contrast

### ISSUE THREAD

**Issue by fflorent** [YunoHost/issues#2614]
## Context

The automatic level badge in README have poor contrast, and even for people having a good sight it can be hard to read:

<img width="213" height="28" alt="Automatic tests result: Level 7" src="https://github.com/user-attachments/assets/a97861ad-fbb4-4e1d-9da6-fce8266cbe90" />

[Level 7 has a ratio of 1.95093](https://contrast-finder.tanaguru.com/result.html?foreground=%23FFFFFF&background=%2397CA00&ratio=4.5&isBackgroundTested=true&algo=Rgb&distanceSort=asc)

<img width="213" height="28" alt="Automatic tests result: Level 8" src="https://github.com/user-attachments/assets/d11cb422-cdf5-475a-9f90-cc642ad38eeb" />

[Level 8 has a ratio of 2.12314](https://contrast-finder.tanaguru.com/result.html?foreground=%23FFFFFF&background=%2344cc11&ratio=4.5&isBackgroundTested=true&algo=Rgb&distanceSort=asc)

## Proposed solution

The text font is large, so we should *at least* aim a ratio of 3:1, ideally 4.5:1 ([source](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Guides/Understanding_WCAG/Perceivable/Color_contrast)).

The links above suggest new contrast ratios.

Please note that this badge has a [very good contrast ratio](https://contrast-finder.tanaguru.com/result.html?foreground=%23FFFFFF&background=%23006400&ratio=4.5&isBackgroundTested=true&algo=Rgb) (7.43924):
<img width="183" height="28" alt="Image" src="https://github.com/user-attachments/assets/17200172-3d22-4129-b7b0-b64fb31268f3" />

We may reuse these color set for one badge or even the two. Wdyt?

**Comment by tituspijean** [comment]
Taken from the aforementioned suggestions, we could change the level background color to, respectively, #5D8210 and #428623.
@fflorent, have you checked the background colors for other levels too? Are these two the only ones with this issue?

**Comment by fflorent** [comment]
Let's see:

## Level badges

### The grey part

Foreground: `#FFFFFF` (approx)
Background: `#555555`

Result: ✅ [7.45518](https://contrast-finder.tanaguru.com/result.html?foreground=%23FFFFFF&background=%23555555&ratio=4.5&isBackgroundTested=true&algo=Rgb), which is pretty good

### The right part (the level themselves)

- Level 8: see above
- Level 7: see above
- Other levels: I have not taken a look, I would appreciate help to get them and will be happy to update this post

## App Demo

<img width="92" height="28" alt="App demo badge" src="https://github.com/user-attachments/assets/b5273f9a-16b0-462d-bf0c-f9904ec6f7df" />

Foreground: `#FFFFFF`
Background: `#007EC6`
Result: [ratio 4.3752](https://contrast-finder.tanaguru.com/result.html?foreground=%23FFFFFF&background=%23007ec6&ratio=4.5&isBackgroundTested=true&algo=Rgb&distanceSort=asc), good enough as the font is bold, could still be slightly improved as suggests the result page.

## Version

<img width="183" height="28" alt="Version badge sample" src="https://github.com/user-attachments/assets/db98608d-95d1-4e69-ac63-0e6d7da17cba" />

### The grey part

Same colorset as for the grey part of the level badges.

### The version itself
Foreground: `#FFFFFF`
Background: `#009600`
Result: [3.91606](https://contrast-finder.tanaguru.com/result.html?foreground=%23FFFFFF&background=%23009600&ratio=4.5&isBackgroundTested=true&algo=Rgb&distanceSort=asc), good enough as the font is bold, but could be easily improved (see the suggestions in the result page)

## "More info on the App Store" / "I found a bug, Report an Issue!"

!["More info on the App store" image](https://github.com/user-attachments/assets/92f84dc9-e42b-443b-b9b1-2a49de1d0838)

Foreground: `#8C6947`
Background: `#FBDDCA`

Result: [ratio 3.85579 ](https://contrast-finder.tanaguru.com/result.html?foreground=%238c6947&background=%23fbddca&ratio=4.5&isBackgroundTested=true&algo=Rgb), could be improved, but would need more investigation. Should be acceptable for the bold part.

**Comment by fflorent** [comment]
I have found the script where the badge are generated:
https://github.com/YunoHost/appstore/blob/7031d98e2cd56122479e54b730babb7f97abb659/assets/badges/fetch_badges
